cypress/integration/configuration/shippingMethods/shippingWeights/shippingWeightsLimits.js

```diff
@@ -23,7 +23,7 @@ import {
 } from "../../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Shipping weight limits", () => {
```

Comment:
This test fails during execution, probably changes to UI were made which resulted in error
![image](https://user-images.githubusercontent.com/97674352/153201496-f47a7fe7-5419-47bf-aa73-fe26fe3d667b.png)